translations/German/general-notes.md

```diff
@@ -0,0 +1,118 @@
+# General Rules
+
+Thank you for considering to contribute to the German translation!
+Before you start writing, please make sure that you have read the
+following general translation rules.
+
+## Formal or Informal?
+
+**Informal.**
+
+Although written German tends to be quite formal, websites in German are
+usually following informal netiquette. As Zulip's guides are written
+in a more colloquial style, German translations should be rather informal as well.
+
+**Don't use slang or regional phrases in the German translation:**
+
+* Instead of *"So'n Dreck kann jedem mal passieren."*, you could
+say *"Dieser Fehler tritt häufiger auf."*
+
+* "Das ist die Seite, wo der Quelltext steht." - the "*wo*" is regional,
+say *"Das ist die Seite, auf der der Quelltext steht."* instead.
+
+## Form of address
+
+**Use "Du" instead of "Sie".**
+
+For the reasons provided in [the previous section](#formal-or-informal),
+stick to *Du* (informal) instead of *Sie* (formal) when addressing
+the reader and remember to capitalize *Du*.
+
+## Form of instruction
+
+**Prefer imperative over constructions with auxiliary verbs.**
+
+For instructions, try to use the imperative (e.g. *"Gehe auf die Seite"* -
+*"Go to the page"*) instead of constructions with auxiliary verbs
+(e.g. *"Du musst auf die Seite ... gehen"* - *"You have to go the page ..."*).
+This keeps the phrases short, less stiff and avoids unnecessary addressings
+of the reader.
+
+## Rules for labels
+
+**Use continuous labels with verbs in infinitive form**
+
+To be consistent with other online platforms, use continuous labels for buttons,
+item titles, etc. with verbs in infinitive form,
+e.g. *Manage streams* - *Kanäle verwalten* instead of *Verwalte Kanäle*.
+
+## Concatenation of words
+
+**Try to avoid it.**
+
+German is famous for its concatenations of nouns
+(e.g. *Heizölrückstoßdämpfung*, which means *fuel oil recoil attenuation*).
+For the sake of correct rendering and simplicity, you should try to avoid such
+concatenations whenever possible, since they can break the layout of the Zulip
+frontend. Try to stick to a maximum length of 20 characters and listen to your
```

Comment:
I think it's `follow your intuition` in English, though a Polish speaker would listen to it too :)